app/controllers/spotlight/contact_images_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Spotlight
+  # Handles requests to upload images for contact images
+  class ContactImagesController < FeaturedImagesController
```

Comment:
Can we accomplish this by routing the appropriate path to the `FeaturedImagesController` and making sure that `avatar_attributes` are handles properly there?